frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -94,7 +94,7 @@ const filters = [{
 export const PersistentVolumeClaimsList = props => <List {...props} Header={Header} Row={Row} />;
 export const PersistentVolumeClaimsPage = props => {
   const createProps = {
-    to: `/k8s/ns/${props.namespace}/persistentvolumeclaims/new/form`,
+    to: `/k8s/ns/${props.namespace || 'default'}/persistentvolumeclaims/new/form`,
```

Comment:
I'm a little hesitant to just use the namespace `default` if you don't have an opportunity to change the namespace in the form. It's almost never what you actually want. We sort of get away with it for the YAML editor since you see it in the YAML and can change it to something else. But you can't do that here.

It would be better to make the namespace something you can pick in the create PVC form.

Reply:
That's true and I know we have been talking about adding the namespace picker in all our create forms. The outcome of that was (I think) that we will do that eventually. I guess we can start with it here, and follow on other create forms ?